cmd/kube-proxy/app/options/options.go

```diff
@@ -57,7 +57,6 @@ func NewProxyConfig() *ProxyServerConfig {
 			ResourceContainer:              "/kube-proxy",
 			IPTablesSyncPeriod:             unversioned.Duration{30 * time.Second},
 			UDPIdleTimeout:                 unversioned.Duration{250 * time.Millisecond},
-			Mode:                           componentconfig.ProxyModeUserspace,
```

Comment:
Ok, it's probably worth adding a BestAvailable ProxyMode [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/types.go#L65). The doc is already wrong.